src/views/settings/p2p-stats/p2p-stats.module.css

```
@@ -0,0 +1,197 @@
+.connectedPeerList {
+  display: grid;
+  gap: 5px;
+  margin-top: 6px;
+}
+
+.connectedPeer {
+  display: grid;
+  gap: 3px;
+  padding: 4px 6px;
+  border: 1px solid rgba(127, 127, 127, 0.28);
+  border: 1px solid color-mix(in srgb, currentColor 22%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)